### PR TITLE
Updated README

### LINKED ISSUE #25: Update Readme
## Summary

* Should conform to new open source template.
  * See the internal repo template project.
* Clear instructions for development and "building".
* For Explorer's Hub... would point to main agent area... don't know what that is anymore. Perhaps via tags? Check with Corey
* Update any out of date information

Depending on the amount of content, may want to run by the docs team. They had a bit of  consistency and branding changes to the main agent readme.

### PR DESCRIPTION
**Pull request by nijotz**
* Updated README
* Closes #25

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,8 +29,54 @@ require('@newrelic/mysql')
...
+
+To run the full suite, run: `npm test`.
+
+Individual test scripts include:
+
+```
+npm run unit
+npm run versioned
+```
+
```
Comment (carlo-808) [inline README.md]: You may want to address how the versioned tests can't run without setting up the mysql docker container from teh main agent.
https://github.com/newrelic/node-newrelic-mysql/issues/25#issuecomment-668878636
Plus, I don't think there are any unit tests that run.  Might want to verify and remove the unit test script from package.json.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,13 @@
-New Relic's official MySQL instrumentation for use with the
-[Node agent](https://github.com/newrelic/node-newrelic). This module is a
-dependency of the agent and is installed with it by running:
+[![Community Project header](https://github.com/newrelic/open-source-office/raw/master/examples/categories/images/Community_Project.png)](https://github.com/newrelic/open-source-office/blob/master/examples/categories/index.md#community-project)
```
Comment (carlo-808) [inline README.md]: I think we need an experimental header link here.
https://github.com/newrelic/node-newrelic-mysql/issues/25#issuecomment-668747382

File: README.md (hunk 1 above)
Comment (nijotz) [reply README.md]: Done and done

File: README.md (hunk 2 above)
Comment (nijotz) [reply README.md]: 👍

**Review by carlo-808 — APPROVED** [review]
⚡ 👍

**Pull request merged**